ubuntu-20.04-focal-arm64v8/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM arm64v8/ubuntu:focal
+
+RUN apt-get -qq update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    cmake \
+    ghostscript \
+    git \
+    libffi-dev \
+    libfreetype6-dev \
+    libfribidi-dev \
+    libharfbuzz-dev \
+    libimagequant-dev \
+    libjpeg-turbo-progs \
+    libjpeg8-dev \
+    liblcms2-dev \
+    libopenjp2-7-dev \
+    libtiff5-dev \
+    libwebp-dev \
+    netpbm \
+    python3-dev \
+    python3-numpy \
+    python3-scipy \
+    python3-setuptools \
+    python3-tk \
+    sudo \
+    tcl8.6-dev \
+    tk8.6-dev \
+    virtualenv \
+    wget \
+    xvfb \
+    zlib1g-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN useradd pillow && \
+    addgroup pillow sudo && \
+    mkdir /home/pillow && \
+    chown pillow:pillow /home/pillow
+
+RUN virtualenv -p /usr/bin/python3.8 --system-site-packages /vpy3 && \
+    /vpy3/bin/pip install --upgrade pip && \
+    /vpy3/bin/pip install cffi olefile pytest pytest-cov && \
```

Comment:
And let's also disable the pip cache to save a bit of extra space, like https://github.com/python-pillow/docker-images/pull/96.